src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,171 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private static final int EMPTY_LIST = 0;
+
+    private Node<T> head;
+
+    private Node<T> tail;
+
+    private int size;
+
+    public MyLinkedList() {
+        this.head = null;
+        this.tail = null;
+        this.size = EMPTY_LIST;
+    }
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value);
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor. Currently, you are only passing the `item` and setting `prev` and `next` to `null` implicitly. It's better to explicitly set these fields in the constructor.